roles/openshift_efk/README.md

```diff
@@ -0,0 +1,9 @@
+###Required vars:
```

Comment:
I would suggest renaming to openshift_logging, and if there are ever alternatives available, then selecting the alternatives can come down to specifying some different variables.